skin/src/main/java/org/researchstack/skin/AppPrefs.java

```diff
@@ -24,11 +24,16 @@
         prefs = PreferenceManager.getDefaultSharedPreferences(context);
     }
 
-    public static synchronized AppPrefs getInstance(Context context)
+    public static void init(Context context) {
```

Comment:
That's a very good point. Should we remove `synchronized` from getInstance since it's not needed, and to make it clearer it's not something we need to worry about in future changes to this class?